api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/BaseDao.java

```diff
@@ -558,8 +558,11 @@ protected Optional<Criterion> handleGender(@Nonnull String propertyName, TokenAn
 		});
 	}
 	
-	protected Optional<Criterion> handleLocationReference(@Nonnull String locationAlias,
-	        ReferenceAndListParam locationReference) {
+	protected Optional<Criterion> handleLocationReference(ReferenceAndListParam locationReference) {
```

Comment:
I needed to support creating a location reference criterion without an alias, see below...